### PR TITLE
Small update in SeedSigner description in wallet.md

### PR DESCRIPTION
**Pull request by marc3linho**
I made some small updates for the SeedSigner signing device.
E.g. price, picture and limitations

### CODE REVIEW

**Comment by BitcoinQnA** [comment]
Fine with all this apart from the removal of 'less approachable for beginners'. That has not changed in my opinion.

**Comment by marc3linho** [comment]
In my opinion it is just "less approchable for beginners" due to the concept of a stateless device.
There are several great beginner guides.
I will add "Less approachable for beginners (due to stateless nature)"

File: wallet/hardware.md [hunk 1]
```diff
@@ -107,9 +107,9 @@ This type of set-up does come with additional hurdles that may be more difficult
...
+[SeedSigner](https://seedsigner.com/) is an offline, air-gapped Bitcoin transaction signing device that can be built from off-the-shelf [components](https://github.com/SeedSigner/seedsigner/blob/main/README.md#shopping-list) for around $70. After gathering and assembling the hardware, users flash the open source [software](https://github.com/SeedSigner/seedsigner/releases) onto a microSD card, insert to the device and power on. SeedSigner is a 'stateless' device, meaning that once powered off, it does not store any private keys. SeedSigner helps users save with Bitcoin by assisting with trustless private key generation and multi-signature wallet setup, and helps users transact with Bitcoin via a secure, air-gapped QR-exchange signing model.
 
 <p align="center">
 <img src="/assets/img/seedsigner(1).png" class=responsive width="410" height="300" maxheight="390">
```
Comment (BitcoinQnA) [inline wallet/hardware.md:115]: Need an image without background to suit the style of the page. 

I like to use a tool like https://www.remove.bg/

File: wallet/hardware.md [hunk 2]
```diff
@@ -119,10 +119,13 @@ This type of set-up does come with additional hurdles that may be more difficult
...
 ### Notable Limitations
-* Less approachable for beginners
-* Less convenient for those transacting more often (due to stateless nature)
+* Less convenient for those transacting more often (*due to stateless nature*)
+* SD card should be removed after startup (*GUI notification since v0.7.0*)
+* SD card should be stored in a secure location
+* Less approachable for beginners (*due to stateless nature*)
 
 ### Other Resources
 * [Codebase](https://github.com/SeedSigner/seedsigner)
```
Comment (BitcoinQnA) [inline wallet/hardware.md:131]: No links to Telegram, video or docs?

Comment (marc3linho) [reply wallet/hardware.md:131]: Links are in Line 132, 133, 134 but unchanged to the original content

File: wallet/hardware.md (hunk 1 above)
Comment (marc3linho) [reply wallet/hardware.md:115]: will provide a better picture soon :+1:

**Comment by BitcoinQnA** [comment]
Sorry for the delay, been pretty busy!

**Pull request merged**